caffe2/python/hypothesis_test_util.py

```diff
@@ -270,6 +270,15 @@ def tensors(n, min_dim=1, max_dim=4, dtype=np.float32, elements=None, **kwargs):
             max_size=n))
 
 
+def ntensors(min_n, max_n, min_dim=1, max_dim=4, dtype=np.float32, elements=None, **kwargs):
+    dims_ = st.lists(dims(**kwargs), min_size=min_dim, max_size=max_dim)
```

Comment:
Please add docstring on what this function is about.

Reply:
updated